packages/web/src/app/cart/cart.component.tsx

```diff
@@ -1,3 +1,4 @@
+// packages/web/src/app/cart/cart.component.tsx
```

Comment:
I would remove this comment, since we never did that previously. Or do it in a separate PR, adding those comments to all our files.